submissions/Jemikalajah-Philip/HTML/Registration.css

```diff
@@ -0,0 +1,116 @@
+*
+{
+    margin: 0px;
+    padding: 0px;
+}
+body
```

Comment:
Please move the curly brace to the same line as the tag, like this:
```css
body {
    background-color: rgb(255, 0, 55);
    font-family: sans-serif;
}
```

You can use this style guide as a reference: https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md

Reply:
Please apply the change to all other CSS rules as well